examples/php/composer.json

```diff
@@ -2,6 +2,6 @@
   "name": "grpc/grpc-demo",
   "description": "gRPC example for PHP",
   "require": {
-    "grpc/grpc": "v1.0.0",
+    "grpc/grpc": "v1.0.0"
```

Comment:
@thinkerou it's syntax error for last property of object and last item of array in json file.